README.md

```diff
@@ -58,6 +59,28 @@ To make sure that the Ruby LSP can find the version manager scripts, make sure t
 configuration script (e.g.: ~/.bashrc, ~/.zshrc) and that the SHELL environment variable is set and pointing to the
 default shell.
 
+##### Custom activation
+
+If you're using a different version manager than the ones listed above or if you're manually inserting the Ruby
+executable into the PATH, you will probably need to define custom activation so that the extension can find the correct
+Ruby.
+
+For these cases, set `rubyLsp.rubyVersionManager` to `"custom"` and then set `rubyLsp.customRubyCommand` to a shell
+command that will activate the right Ruby version or simply add the Ruby bin folder to the PATH. Some examples:
+
+```jsonc
+{
+  // Don't forget to set the manager to custom when using this option
+  "rubyLsp.rubyVersionManager": "custom",
+
+  // Using a different version manager than the ones included by default
+  "rubyLsp.customRubyCommand": "my_custom_version_manager activate",
+
+  // Adding a custom Ruby bin folder to the PATH
+  "rubyLsp.customRubyCommand": "PATH=/path/to/ruby/bin:$PATH"
```

Comment:
btw on Windows this may need `;` instead of `:`.

Reply:
Yeah, users will need to write their own activation based on their machines.